### DISCUSSION TITLE
Getting the API Management menu to appear

### DISCUSSION THREAD

**Discussion by Alwaysin** [FreshRSS/FreshRSS#6454] [Q&A] [answered]
It's been a long time I haven't used the API, I wanted to try again an Android app but I cannot set an API password, I don't have "API Management" menu displaying:

![chrome_WHysY4wjHB](https://github.com/FreshRSS/FreshRSS/assets/7189936/cd698966-0404-4916-971e-0c1ac4e98eff)

How could I try to debug this? I haven't seen errors on browser console nor web server logs.

**Comment by Frenzie** [comment, marked as answer]
It's under profile.

**Reply by Alwaysin** [reply]
I swear I looked everywhere... except here I guess! Thank you!

**Reply by math-GH** [reply]
the documentation helps too :)

https://freshrss.github.io/FreshRSS/en/users/06_Mobile_access.html#enable-the-api-in-freshrss
